src/common/utils/string.js

```diff
@@ -6,10 +6,10 @@ export function formatActiveUserFirstNames(users) {
     : `${activeUserFirstNames.slice(0, -1).join(', ')} and ${activeUserFirstNames.slice(-1)}`
 }
 
-export function getCommiterLabel(userCount, capitalizeIt = false) {
-  const label = userCount < 2 ? 'commiter' : (userCount === 2 ? 'pair' : 'mob')
+export function getNotificationLabel(userCount, shouldCapitalize = false) {
```

Comment:
What if we always capitalized and got rid of the `shouldCapitalize` argument? Seems appropriate given the two usages of this function. Also we could potentially use lodash for capitalization here rather than our custom function.

Reply:
Yeah I would vote for pulling out the `shouldCapitialize` and just let the consumer use lodash's `capitalize` function itself.

Reply:
I'm like the idea of pulling in `lodash.capitalize`, but I think the `shouldCapitlaize` param is still useful because one of the usages of `getNotificationLabel` is in a sentence rather than as a title (I think the initial notification we post is that way).

Reply:
The initial notification we post is the name of the current committer. Which should probably always be capitalized, right?

Reply:
That function just returns "author" for a single user, "pair" for two, and "mob" for more. Sometimes the word is used at the beginning of a title or sentence so it needs to be capitalized and sometimes it's in the middle of the title or sentence so it shouldn't. If there are only two usages of it we could probably pull the capitalization logic out and just use lodash's capitalize in the consuming function. But I don't feel super strong about that as long as it stays capitalized where it needs to be.

Reply:
Right. I'm good with either. I have a commit ready to add lodash capitalize.